### PR TITLE
OCT-1675: E2E: App wide fiat support - allocation view

### PR DESCRIPTION
**Pull request by jmikolajczyk**
## Definition of Done

1. [ ] Acceptance criteria are met.
2. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
3. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
4. [ ] Unit tests are added unless there is a reason to omit them.
5. [ ] Automated tests are added when required.
6. [ ] The code is merged.
7. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
8. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/cypress/e2e/allocationItemWindowOpen.cy.ts [hunk 1]
```diff
@@ -141,6 +152,60 @@ describe('allocation (allocation window open)', () => {
...
+          .type('0.002');
+        cy.get('[data-test=AllocationRewardsBox__section__value--0]')
+          .invoke('text')
+          .should('eq', '0.007 ETH');
+        cy.get('[data-test=AllocationRewardsBox__section__value--1]')
+          .invoke('text')
+          .should('eq', '0.003 ETH');
+      });
+
+      it('User can change allocation item value manually ("Use crypto as main value display": false)', () => {
```
Comment (aziolek) [inline client/cypress/e2e/allocationItemWindowOpen.cy.ts:189]: I think using variables from the store or key from translations (`cryptoMainValueDisplay`) is more universal and would enforce being in sync across the codebase. Imagine changing this string in `translations.json` -- you probably won't take a look into the CY. WDYT?
```suggestion
      it(`User can change allocation item value manually (${IS_CRYPTO_MAIN_VALUE_DISPLAY}: false)`, () => {
```

File: client/src/components/Allocation/AllocationNavigation/AllocationNavigation.tsx [hunk 2]
```diff
@@ -41,9 +41,15 @@ const AllocationNavigation: FC<AllocationNavigationProps> = ({
 
   return (
     <div className={styles.root}>
-      <Button className={styles.button} {...commonProps} {...buttonPreviousProps} />
       <Button
         className={styles.button}
+        {...commonProps}
+        {...buttonPreviousProps}
+        dataTest="AllocationNavigation__previousButton"
```
Comment (aziolek) [inline client/src/components/Allocation/AllocationNavigation/AllocationNavigation.tsx:48]:
```suggestion
        dataTest="AllocationNavigation__Button--previous"
```

File: client/src/components/Allocation/AllocationNavigation/AllocationNavigation.tsx [hunk 3]
```diff
@@ -41,9 +41,15 @@ const AllocationNavigation: FC<AllocationNavigationProps> = ({
...
+      />
+      <Button
+        className={styles.button}
+        dataTest="AllocationNavigation__nextButton"
```
Comment (aziolek) [inline client/src/components/Allocation/AllocationNavigation/AllocationNavigation.tsx:52]:
```suggestion
        dataTest="AllocationNavigation__Button--next"
```

Comment (aziolek) [inline client/src/components/Allocation/AllocationNavigation/AllocationNavigation.tsx:52]:
```suggestion
        dataTest="AllocationNavigation__Button--next"
```

**Review by aziolek — CHANGES_REQUESTED** [review]

File: client/cypress/e2e/allocationItemWindowOpen.cy.ts (hunk 1 above)
Comment (jmikolajczyk) [reply client/cypress/e2e/allocationItemWindowOpen.cy.ts]: Good idea.

**Review by aziolek — APPROVED** [review]

**Pull request merged**